dev-proxy-plugins/MockResponses/MockResponse.cs

```diff
@@ -30,5 +30,5 @@ public class MockResponseResponse
     [JsonPropertyName("body")]
     public dynamic? Body { get; set; }
     [JsonPropertyName("headers")]
-    public IDictionary<string, string>? Headers { get; set; }
+    public List<IDictionary<string, string>>? ResponseHeaders { get; set; }
```

Comment:
Let's keep `Headers`, so that we don't repeat `response` in `Response.ResponseHeaders`